appveyor.yml

```diff
@@ -1,9 +1,11 @@
 version: 1.0.{build}
 
+# Operating system (build VM template)
+os: Visual Studio 2015 CTP
+
 environment:
   matrix:
-#   - VS_VERSION: MSVC2012
-    - VS_VERSION: MSVC2013
```

Comment:
@pksunkara Matrix, as far as understand it, is for environment variables (eg. debug mode or release). You would need different OS images for having actual different versions of Visual Studios (eg Windows Server has MSVS2012 "OS" and and MSV2015 "OS" image has MSVS15... 

I cannot afford exploring various images setups in AppVeyor at the moment. Maybe something we can put into sprint. 

Keep in mind that `VS_VERSION: MSVC2013` is just a env variable that is used only by our GYP tool (= it does not control the AppVeyor environment in any way)